src/Classes/Particle.ts

```diff
@@ -74,7 +78,9 @@ export class Particle {
             this.rotateDirection = options.particles.rotate.direction;
         }
 
-        this.radius = (options.particles.size.random ? Math.random() : 1) * container.retina.sizeValue;
+        const randomSize = options.particles.size.random as IRandomSize;
+
+        this.radius = randomSize.enable ? Utils.randomInRange(randomSize.minimumValue, container.retina.sizeValue) : container.retina.sizeValue;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Exceeds maximum line length of 120](https://app.codacy.com/manual/ar3s/tsparticles/pullRequest?prid=5216054)